components/intake.py

```diff
@@ -30,7 +30,7 @@ def execute(self):
         """Run at the end of every control loop iteration."""
         pass
 
-    def intake_rotate(self, value):
+    def rotate_intake(self, value):
```

Comment:
Don't I already know that I'm going to rotate the intake if I ask the intake to rotate?